### DISCUSSION TITLE
Challenge 2 - Interactable Map of LA Examining Cost Burden Among Renters in 2022 by Census Tract & Ethinicity

### DISCUSSION THREAD

**Discussion by KathrynMercer** [datakind/datakit-housing-fall-2024#23] [Show and tell]
TLDR: See zip file for interactable map of cost-burdened renters + Jupyter notebook (feel free to use/modify code for other mapping applications!) [LA + Orange Counties CostBurden + Ethnicity.zip](https://github.com/user-attachments/files/17094905/LA.%2B.Orange.Counties.CostBurden.%2B.Ethnicity.zip)

What is it: An interactable map of the census tracts of LA & Orange Counties with choropleth mapping of cost-burdened renters (ie. renters who spent more than 30% of their income on rent monthly) for the 2022 year. 

What data was used: (all referenced from the US census.gov website)
- American Community Survey 2022 (5 year estimates) for Housing Costs (AKA B25070)
- American Community Survey 2022 (5 year estimates) for Demographic Data (AKA DP05)
- TIGER/Line Shapefile of CA from 2022 

What tools were used: 
- Jupyter notebook w/ Python
- Pandas, GeoPandas, and Folium libraries (also MatPlotLib for the static maps that are found in the notebook)

Limitations of the data:
- 5 Year Estimates are released yearly but include collection data from the preceding 5 years, which means mapping change over time (ex. to examine the COVID rent moratoriums) is not ideal as the map changes will likely lag the real-world changes significantly. 
- Inconsistent sampling - some of the census tracts have relatively few renters sampled, which leads to potentially nonrepresentative proportions (which is what the choropleth is based on). So if the census tract has a dramatic proportion of renters being cost-burdened (ie. 1), be sure to check that there wasn't a very small number of renters in that census tract's sample. Alas, this is the data. 

What does it show:
- Areas of cost-burdened renters seem to concentrate around urban areas and (eyeballing it) State Route 99 on the full map of CA (which was too large to easily share)
- Ethnicity seems to match broadly with urban/cost-burdened areas, though it's harder to see this on a specific census-tract level due to limitations of the data & visualization (discussed below). 

Next Steps: 
- Ethnicity - I'm not yet satisfied with my ethnicity map layer. It currently shows only proportion of residents (not renters) of different ethnicities by geographic areas. The color gradient on the ethnicity layer expresses people of color generally without the granularity of specific races (thought the mouseover data will show more specific information). I'd like to modify this to be more granular, probably  across multiple layers or maybe with Multiselect boxes (AKA check boxes) or radio buttons. 
- Additional demographic/home information - I'd like to expand the layers to show ages, sex, income, and housing attributes (ex. does median rent correlate with cost-burden proportion?)
- Homeowner expansion - my current data is only for renters because that's the data I found first. ACS has a ton of surveys, so I'm sure they have a homeowner version that should be easy enough to integrate to this visualization (famous last words!)
- Time Series - I would like to do a visualization to examine the effect of COVID rent moratoriums on the cost-burdened proportions, unfortunately, this data won't work for that. We should consider tracking down more data (perhaps via California Department of Housing and Community Development (HCD) or National Low Income Housing Coalition (NLIHC) Out of Reach reports).

**Comment by kodiakpony** [comment]
This is very cool!  I'm looking to get started using GeoPandas and Folium for the first time, so this is very helpful to me.
